spaceship/lib/spaceship/client.rb

```diff
@@ -545,6 +552,21 @@ def send_shared_login_request(user, password)
     end
     # rubocop:enable Metrics/PerceivedComplexity
 
+    def fetch_hashcash
+      response = request(:get, "https://idmsa.apple.com/appleauth/auth/signin?widgetKey=#{self.itc_service_key}")
+      headers = response.headers
+
+      bits = headers["x-apple-hc-bits"]
+      challenge = headers["x-apple-hc-challenge"]
```

Comment:
```suggestion
      bits = headers["X-Apple-HC-Bits"]
      challenge = headers["X-Apple-HC-Challenge"]
```

Reply:
Any particular reason why this was written in lowercase, and the one above was all uppercase @joshdholtz ? 👀

Reply:
It's what the hash showed when I inspected it but I guess it does a case insensitive lookup 😇

Reply:
Essentially HTTP header names are case-**in**sensitive so the case doesn't really matter. It's only a styling concern🙂 

https://stackoverflow.com/questions/5258977/are-http-headers-case-sensitive